app/config/sql/board1_1.sql

```diff
@@ -0,0 +1,105 @@
+--
+-- Create database
+--
+DROP DATABASE board1_1;
+CREATE DATABASE board1_1;
+
+--
+-- Assume that the user have been created.
+--
+GRANT SELECT, INSERT, UPDATE, DELETE ON board1_1.* TO board_root@localhost;
+FLUSH PRIVILEGES;
+USE board1_1;
+
+--
+-- Create tables
+--
+
+CREATE TABLE category (
+    id          TINYINT UNSIGNED NOT NULL AUTO_INCREMENT,
+    name        VARCHAR(20),
+    PRIMARY KEY (id)
+) ENGINE=InnoDB;
+
+CREATE TABLE user (
+    id          INT UNSIGNED NOT NULL AUTO_INCREMENT,
+    username    VARCHAR(16) NOT NULL UNIQUE,
+    first_name  VARCHAR(30) NOT NULL,
+    last_name   VARCHAR(30) NOT NULL,
+    email       VARCHAR(30) NOT NULL UNIQUE,
+    password    VARCHAR(60),
```

Comment:
I think there should be `NOT NULL` here po unless password is not required. :bow: